includes/services/Leaflet/leaflet-providers/leaflet-providers.js

```diff
@@ -640,17 +742,19 @@
 				// NB : 'choisirgeoportail' is a demonstration key that comes with no guarantee
 				apikey: 'choisirgeoportail',
 				format: 'image/jpeg',
+				style : 'normal',
 				variant: 'GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN-EXPRESS.STANDARD'
 			},
 			variants: {
 				parcels: {
 					options : {
 						variant: 'CADASTRALPARCELS.PARCELS',
 						maxZoom: 20,
+						style : 'bdparcellaire',
 						format: 'image/png'
 					}
 				},
-				ign_maps: 'GEOGRAPHICALGRIDSYSTEMS.MAPS',
+				ignMaps: 'GEOGRAPHICALGRIDSYSTEMS.MAPS',
```

Comment:
@JeroenDeDauw this change should be also made in Maps_Settings.php#379 : 

`'GeoportailFrance.ign_maps' => true,`

should be replaced by : 

`'GeoportailFrance.ignMaps' => true,`